docs/UserGuide.md

```diff
@@ -164,23 +166,61 @@ A member of our core team will attend to you as soon as possible.
 
 ## User Interface
 
-{to be filled in}
+`EduMate`'s UI components are built using FXML and JavaFX. Each of the UI components are built to fulfil a specific functionality. We also used light purple and white as our main colour scheme as they are cool colours.
+
+![Ui](images/UG_ui_pictures/user_interface.png)
+<center>Diagram 2: <code>EduMate</code>'s UI</center>
 
 ### Person List
 
-{to be filled in}
+You may locate the Person List at the center-left of `EduMate`. This is where you will see all your saved contacts (or sample data if you are trying out for the first time).
```

Comment:
I think instead of showing the individual sections one by one, just show the UI as a whole but with boxing and numbering, then use those to link to the different sections. Might be neater to the eye. I can do this if needed.